pkg/config/process_test.go

```diff
@@ -151,6 +152,12 @@ func TestEnvVarOverride(t *testing.T) {
 			value:    "1h",
 			expected: time.Hour,
 		},
+		{
+			key:      "process_config.disable_realtime_checks",
+			env:      "DD_PROCESS_CONFIG_DISABLE_REALTIME_CHECKS",
+			value:    "true",
+			expected: true,
+		},
```

Comment:
do we also want to test the Yaml override?

https://github.com/DataDog/datadog-agent/blob/c71c34cd412f41042c5c31bbaedc00a50088a5ea/pkg/process/config/config_test.go#L170-L178

Reply:
I'd like to avoid introducing more tests for basic configs to `pkg/process/config` - all the custom logic from there is going to be removed.